jina/__init__.py

```diff
@@ -39,13 +39,18 @@ def _warning_on_one_line(message, category, filename, lineno, *args, **kwargs):
 
 # JINA_MP_START_METHOD has higher priority than os-patch
 _start_method = _os.environ.get('JINA_MP_START_METHOD', None)
-
 if _start_method and _start_method.lower() in {'fork', 'spawn', 'forkserver'}:
     from multiprocessing import set_start_method as _set_start_method
 
-    _set_start_method(_start_method.lower())
-    _warnings.warn(f'multiprocessing start method is set to `{_start_method.lower()}`')
-    _os.environ.pop('JINA_MP_START_METHOD')
+    try:
+        _set_start_method(_start_method.lower())
+        _warnings.warn(
+            f'multiprocessing start method is set to `{_start_method.lower()}`'
+        )
+    except Exception as e:
+        _warnings.warn(
```

Comment:
Please guide the user to say that most likely it was already set

Reply:
the exception `{e!r}` explains the context is already set